modules/rest-api/pages/rest-xdcr-identify-target-cluster.adoc

```diff
@@ -0,0 +1,180 @@
+| The provided username has insufficient privileges to call this method.
+
+| `404 Object Not Found`
+| Error in the URI path.
+
+| `500 Could not retrieve the requested repository`
+| Error in Couchbase Server.
+
+|===
+
+== Examples
+
+The following examples demonstrate how to use the `xdcr/sourceClusters` API to identify the target cluster in a replication and inspect the incoming replication streams.
+
+[#example-source-without-replication-specs]
+=== Cluster without replication specifications
+
+----
+curl -X GET -u Administrator:pw http://127.0.0.1:9000/xdcr/sourceClusters | jq
+----
+
+This call retrieves the following details of a source cluster with one node.
+This source cluster has created a remote cluster reference to a target cluster but has not yet created a replication specification.
+
+----
+[
+  {
+    "SourceClusterNodes": [
+      "127.0.0.1:9002"
+    ],
+    "SourceClusterReplSpecs": null,
+    "SourceClusterUUID": "d4b15b1069e4aaac1d6d0cb7d15b1dde",
+    "SourceClusterName": “m1”,
+    "SourceClusterHBReceiveTime": "2024-10-24T20:52:21.902848+05:30",
+    "SourceClusterHBExpiryTime": "2024-10-24T21:02:21.902848+05:30"
+  }
+]
+----
+
+[#example-source-with-replication-specs]
+=== Cluster with replication specifications
+
+----
+curl -X GET -u Administrator:pw http://127.0.0.1:9002/xdcr/sourceClusters | jq
+----
+
+This source cluster has two nodes, has a reference established with the target cluster, and is replicating to the target cluster.
+This call retrieves source cluster information and its replication specifications to the target cluster.
+
+----
+[
+  {
+    "SourceClusterNodes": [
+      "10.100.172.24:9000",
+      "127.0.0.1:9001"
+    ],
+    "SourceClusterReplSpecs": [
+      {
+        "id": "d4b15b1069e4aaac1d6d0cb7d15b1dde/B0/B1",
+        "internalId": "ZsrIjhmEaf9232WGyoSexg==",
+        "sourceBucketName": "B0",
+        "sourceBucketUUID": "64b784ca2b3e8df7ab1cbf70ab89772f",
+        "targetClusterUUID": "d4b15b1069e4aaac1d6d0cb7d15b1dde",
+        "targetBucketName": "B1",
+        "targetBucketUUID": "1d86409fe13ffbeb36ba3bf382005480",
+        "replicationSettings": {
+          "values": {
+            "CollectionsMgtMulti": 8,
+            "active": true,
+          ....
+          "Revision": null
+        },
+        "Revision": "g2gCYQBsAAAAAWgCbQAAACA2OGRiNmU0ZDE0YWQwNTFlODM3ZDM1N2FjYTYxMGIzZWgCYQNuBQAUvLneDmo="
+      }
+    ],
+    "SourceClusterUUID": "6aa2a7cc7d9a42c253e76f832055ccea",
+    "SourceClusterName": “m2”,
+    "SourceClusterHBReceiveTime": "2024-10-24T20:52:21.902848+05:30",
+    "SourceClusterHBExpiryTime": "2024-10-24T21:02:21.902848+05:30"
+  }
+]
+----
+
+[#complete-response]
+=== Cluster with replication specifications - a full example
+
+----
+curl -s -X GET -u Administrator:password http://localhost:8091/xdcr/sourceClusters | jq
+----
+This call retrieves the source cluster information and its replication specifications to the target cluster.
```

Comment:
This call retrieves information for all incoming replications into the cluster.  The example output shows that there are two different remote clusters, one called "South" and one called "East", replicating data to this cluster.  The "South" cluster is replicating data from two different buckets, and "North" cluster is replicating data from one bucket.  The information from this REST API call is the same information displayed in the UI for XDCR Incoming Replications.